src/styles/styles/card.scss

```diff
@@ -0,0 +1,147 @@
+.catalog__content {
+  box-sizing: border-box;
+  display: grid;
+  grid-template-rows: auto;
+  grid-template-columns: 200px;
+
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 50px 40px ;
+  margin: 0 auto;
+
+  @media (min-width: 488px) {
+    grid-template-columns: 200px 200px;
```

Comment:
you can create a variable for this 200px which you write several times (like $card-width)